tutorials/commands/ssh-across-clusters.rst

```diff
@@ -0,0 +1,156 @@
+.. _ssh-across-clusters:
+
+===================
+SSH across clusters
+===================
+
+Let's say you want to create a Flux instance in an allocation on a cluster (e.g., let's say out first cluster is "noodle") 🍜️
+and then connect to it via ssh from another cluster (let's say our second cluster is called quartz). This is possible with the right
```

Comment:
should quartz be in quotes too? like noodle?

Reply:
sure, that would be consistent!